src/pages/references/language-clarity-types.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Clarity Language
+description: See a detailed list of all keywords and functions for the Clarity language.
+---
+
+## Clarity Type System
```

Comment:
@lgalabru @kantai - can this part also be generated?

Reply:
It could be, it would require some work to produce a types JSON from the stacks-blockchain node, but it's possible.

Reply:
is this something we could take on as part of the docs week / soon? I created an [issue](https://github.com/blockstack/docs/issues/873) to discuss further